### ISSUE TITLE
The Seeds provided for database setup are not allied with the Mongoose Model

### ISSUE THREAD

**Issue by Swap76** [prabhuomkar/frisky#9]
In the seeds, the service provider in the show is an array of object with properties as name and ID

<img width="188" alt="Screenshot 2019-11-06 at 3 23 25 PM" src="https://user-images.githubusercontent.com/37082605/68287878-74f66800-00a9-11ea-9cf7-5719619071e0.png">

But the model suggests it should be an array of names of service providers

<img width="183" alt="Screenshot 2019-11-06 at 3 20 47 PM" src="https://user-images.githubusercontent.com/37082605/68287990-a707ca00-00a9-11ea-8816-b6e677295568.png">

In short, the seeds should be changed to follow the same convention

**Comment by Swap76** [comment]
@prabhuomkar I will love to send a PR for the solving the issue :-)

**Comment by prabhuomkar** [comment]
@Swap76 yeah send a PR ASAP!
